lib/Fullness/Fullness.cpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file Fullness.cpp
+ * @brief C file for Fullness class
+ * @version 0.1
+ * @date 2022-05-04
+ *
+ */
```

Comment:
You can omit the file comments on source files. You only need them on header files.